diff --git a/zshmarks.plugin.zsh b/zshmarks.plugin.zsh
index 7d2ab5f..d9a4ed6 100644
--- a/zshmarks.plugin.zsh
+++ b/zshmarks.plugin.zsh
@@ -105,7 +105,7 @@ function showmarks() {
       bookmark_path="${bookmark_line%%|*}"
       bookmark_path="${bookmark_path/\$HOME/~}"
       bookmark_name="${bookmark_line#*|}"
-      printf "%s\t\t%s\n" "$bookmark_name" "$bookmark_path"
+      printf "%32s: %s\n" "$bookmark_name" "$bookmark_path"
     done
   fi
 }
